CHANGELOG.rst

```diff
@@ -1,42 +1,54 @@
-# Changelog
+Changelog
+==========
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-### Added
+[Unreleased]
+==============
+Added
+--------
 - Getting and plot integrated intensity now support signals containing nan (#722)
 - Add Symmetry1D signal class and symmetry analysis methods (#724)
 - BeamShift class, which includes the `make_linear_plane` method for better correction of the beam shift when scanning large regions in STEM (#746)
 - Add unit testing of docstring examples (#766)
 
-### Removed
+Removed
+--------
 - lazy_* virtual imaging has been removed, use get_integrated_intensity (#722)
 
-## 2021-04-14 - version 0.13.2
-### Added
- - Code now support python 3.9 
+2021-04-14 - version 0.13.2
+=============================
+Added
+-------
+ - Code now support python 3.9
  - Code now runs on hyperspy 1.6.2
-### Fixed 
+Fixed
```

Comment:
Newlines between sections. Number of underlining === and --- shouldn't be more than the header span.